docs/resolvers/custom-resolvers.md

```diff
@@ -100,9 +100,54 @@ await context.Note.findBy(args, context);
 
 ### info
 
-The `GraphQLResolveInfo` object is available to every resolver and contains information about the GraphQL schema and the current operation.
+The optional `GraphQLResolveInfo` object is available to every resolver and contains information about the GraphQL schema and the current operation.
 You can optionally pass this to the `GraphbackCRUDService` method. A common use for this parameter is mapping the selected fields from the GraphQL query to the database query. 
 
 ```ts
 await context.Note.findBy(args, context, info);
-```
\ No newline at end of file
+```
+
+:::info
```

Comment:
It is strange to see such a large `info` block like this, these are usually reserved for small tidbits of information

![image](https://user-images.githubusercontent.com/11743717/93463793-2bd58e00-f8e0-11ea-9559-a090582ba17e.png)

Reply:
I agree with you. I can make it a sub section. WDYT?

Reply:
Yep that would be best.